server/src/main.py

```diff
@@ -2,7 +2,21 @@
 
 import uvicorn
 
+def InjectSecret(secret_name:str):
```

Comment:
Let's stick to python naming conventions: functions in snake_case and private (file-scoped) functions start with underscore. So these would be _inject_secrets and _inject_secret)

Also something that will get caught by our linter later, type hints are done with a space after the colon.